macros/PreviousMenuNext.ejs

```diff
@@ -80,19 +57,18 @@ if ($2) {
   var linkText = linkTextArray[linkTextArray.length-1];
   var re = /_/gi;
   var finalString = linkText.replace(re,' ');
-  
+
   if($0 && $1) {
-    strMenu = '<a style="float:' + prevFloatValue + '; width: 60%;text-align: center;" href="/' + lang + '/docs/' + string.replace($2, ' ', '_') + '"><span style="' + buttonStyle + '"><i class="icon-arrow-up" aria-hidden="true"></i>' + s_Menu + finalString + '</a>';
+    strMenu = '<a href="/' + lang + '/docs/' + string.replace($2, ' ', '_') + '"><span><i class="icon-arrow-up" aria-hidden="true"></i>' + s_Menu + finalString + '</a>';
   } else if($0) {
-    strMenu = '<a style="float:' + nextFloatValue + '; width: 60%;text-align: ' + nextTextAlign + ';" href="/' + lang + '/docs/' + string.replace($2, ' ', '_') + '"><span style="' + buttonStyle + '"><i class="icon-arrow-up" aria-hidden="true"></i>' + s_Menu + finalString + '</a>';
+    strMenu = '<a href="/' + lang + '/docs/' + string.replace($2, ' ', '_') + '"><span><i class="icon-arrow-up" aria-hidden="true"></i>' + s_Menu + finalString + '</a>';
   } else if($1) {
-    strMenu = '<a style="float:' + prevFloatValue + '; width: 60%;text-align: ' + prevTextAlign + ';" href="/' + lang + '/docs/' + string.replace($2, ' ', '_') + '"><span style="' + buttonStyle + '"><i class="icon-arrow-up" aria-hidden="true"></i>' + s_Menu + finalString + '</a>';
+    strMenu = '<a href="/' + lang + '/docs/' + string.replace($2, ' ', '_') + '"><span><i class="icon-arrow-up" aria-hidden="true"></i>' + s_Menu + finalString + '</a>';
```

Comment:
I am not 100%(this is a little hard to read 😸), but it seems these first `span` elements are not needed?

```suggestion
    strMenu = '<a href="/' + lang + '/docs/' + string.replace($2, ' ', '_') + '"><i class="icon-arrow-up" aria-hidden="true"></i>' + s_Menu + finalString + '</a>';
```

Reply:
Updated! Thanks for catching this!